packages/forms-web-app/src/pages/projects/register/_common/complete/controller.js

```diff
@@ -18,7 +19,7 @@ const getRegisterCompleteController = (req, res) => {
 			...viewModel[key],
 			ipRefNo,
 			email,
-			nsipProjectLink: nsipProjectLink(req.session.appData)
```

Comment:
Having a quick glance of the codebase - it looks like this is the only place the 'nsipProjectLink' function is being called. Might be a good opportunity to raise a tech debt ticket to investigate if it is redundant and if so, to remove it.

Reply:
Thanks, will do.